aidants_connect_web/urls.py

```diff
@@ -59,8 +62,10 @@
     path("logout/", service.logout_page, name="logout"),
     path("select_demarche/", id_provider.fi_select_demarche, name="fi_select_demarche"),
     # FC_as_FS
-    path("fc_authorize/", FC_as_FS.fc_authorize, name="fc_authorize"),
+    path("fc_authorize/<str:source>/", FC_as_FS.fc_authorize, name="fc_authorize"),
     path("callback/", FC_as_FS.fc_callback, name="fc_callback"),
+    # usager: mandats
+    path("espace-usager", espace_usager.espace_usager_home, name="espace_usager_home"),
```

Comment:
Nitpick : par souci de cohérence avec les autres patterns définis, je proposerais 
```suggestion
    path("espace_usager/", espace_usager.espace_usager_home, name="espace_usager_home"),
```

Reply:
fixed. même si je suis plutôt dans le camp `-` (vs `_`) et changer toutes les autres url instead 🙃

Reply:
(je suis dans le même camp :p)